test/integration/node-specific/abstract_cursor.test.ts

```diff
@@ -1,114 +1,263 @@
 import { expect } from 'chai';
+import { once } from 'events';
+import * as sinon from 'sinon';
 import { inspect } from 'util';
 
-import { type Collection, MongoAPIError, type MongoClient } from '../../mongodb';
-
-const falseyValues = [0, 0n, NaN, '', false, undefined];
+import { type Collection, type FindCursor, MongoAPIError, type MongoClient } from '../../mongodb';
 
 describe('class AbstractCursor', function () {
-  let client: MongoClient;
+  describe('regression tests NODE-5372', function () {
```

Comment:
We can mention the ticket in a comment but can we make the title about not double invoking transform